src/cachingreader.cpp

```diff
@@ -246,40 +267,20 @@ void CachingReader::process() {
                 qDebug() << "ERROR: status.chunk is NULL in CHUNK_READ_SUCCESS ReaderStatusUpdate. Ignoring update.";
                 continue;
```

Comment:
I don't think it can happen currently unless there's a bug. Skipping the ReaderStatusUpdate (continue) is the only thing I can think of for handling this.